packages/cli/index.ts

```diff
@@ -569,9 +559,7 @@ async function writeDeps(
     hijackDeps(realDeps, pJson.peerDependencies);
   }
 
-  await writePackageJSON(pJsonPath, pJson);
```

Comment:
this seems like a mistake? it means `writeDeps` will no longer write the `package.json`, which is the whole point of this function 😬 

the line below is a _restore_ function, i.e. it shouldn't be doing any reformatting as its meant to restore the original raw contents

this line is meant to write the formatted _new_ JSON